tests/src/test/scala/cats/tests/TraverseSuite.scala

```diff
@@ -32,6 +32,12 @@ abstract class TraverseSuite[F[_]: Traverse](name: String)(implicit ArbFInt: Arb
     }
   }
 
+  test(s"Traverse[$name].traverseMatches_ with Option") {
```

Comment:
did you mean to say `traverse_ matches traverse with Option`?

Reply:
good catch, thanks. Will you take a look again?